robot_smach_states/src/robot_smach_states/navigation/constraint_functions/arms_reach_constraints.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+# System
+import math
+
+# ROS
+import rospy
+from geometry_msgs.msg import Point
+
+# TU/e Robotics
+from cb_planner_msgs_srvs.msg import OrientationConstraint, PositionConstraint
+
+
+def arms_reach_constraint(pose_designator, arm_designator, look=True):
+    """
+    Position so that the arm can reach the position/entity
+    :param pose_designator: designator that resolves to a FrameStamped
+    :param arm_designator: which arm to use for manipulation
+    :param look: whether or not the orientation must be constrained as well
+    :return: tuple(PositionConstraint, OrientationConstraint)
+    """
+    arm = arm_designator.resolve()
+    if not arm:
+        rospy.logerr("Could not resolve arm")
+        return None
```

Comment:
The type contract of a Designator is actually that: if it can't find an answer, it returns `None`

Reply:
I think Albert is referring to the function itself, which returns None here rather than a tuple of constraints. I already adjusted the type contract to reflect this